### DISCUSSION TITLE
Having Instagram like fullscreen event in FlashList

### DISCUSSION THREAD

**Discussion by Bayramito** [Shopify/flash-list#999] [General]
Is it possible to have such effect like in instagram's reels list?

https://github.com/Shopify/flash-list/assets/44513402/98366e36-072f-4915-83f6-363ca4e779f0

As you can see the videos starting to play in the list, and when you click them they translating to fullscreen mode while continues to play the video. 

Is it possible to do that in Flashlist. at all ? or how they do that? and ideas ?

**Comment by MrGainn** [comment]
Did you figure it out?

**Comment by CavalcanteLeo** [comment]
> Is it possible to have such effect like in instagram's reels list?
> 
>  RPReplay_Final1702626410.MP4 
> As you can see the videos starting to play in the list, and when you click them they translating to fullscreen mode while continues to play the video.
> 
> Is it possible to do that in Flashlist. at all ? or how they do that? and ideas ?

dow did u go with this?

**Reply by MrGainn** [reply]
https://github.com/iNerdStack/react-native-flexible-grid

Works perfect!
